docs/basic/getting-started/hooks.md

```diff
@@ -150,7 +150,7 @@ function Foo() {
 
   useEffect(() => {
     // Note that ref.current may be null. This is expected, because you may
-    // conditionally render the ref-ed element, or you may forgot to assign it
+    // conditionally render the ref-ed element, or you may forget to assign it
```

Comment:
At this point `divRef.current` is null if you **forgot** to assign it. This is not a typo